proposals/nixpkgs-security.md

```diff
@@ -0,0 +1,278 @@
+
+- Total: 30K EUR
+
+Explaination:
+
+The working mode we envision is a strong web developer with extensive Nix experience developing a reusable high standard platform with
+maintenance in mind, i.e. end to end testing, etc, which has a slight amount of flexibility to be reused for similar Linux distributions, e.g. Guix.
+
+We decide to split the work in 3 "big milestones" or "target goals" under the User story item where we want the platform to be usable and acceptable for that target audience, i.e. security team, maintainers and end-users, each phase comes with its lots of features and should work closely with the target audience to ensure we build something of use to that particular audience on the basis of UX/UI research methods. Each phase includes a user discovery part and a development part, user discovery can be performed asynchronously in some situations to improve the velocity.
+
+Label tracker is an important piece of the puzzle for end-users as a technical API for untangling the complicated relations with the large continuous integration of NixOS, our Git repositories, our channels to answer non-trivial questions like "Is this PR in my distribution channel?" or "When is this PR expected to land in my distribution channel?" so we can depend critically on it, those improvements can include performance work, scaling up the infrastructure to be a common good in the Nix community open to everyone.
+
+---
+
+Overall, Nix maintainers essentially have to buy their employers’ time or take compensation that can compete with other engagements; known contributors who could do the bulk of the work are in a similar situation or otherwise would need substantial time for onboarding. Therefore we have to assume consultancy rates to be able to keep to the time frame, with the option to trade budget for velocity to some extent.
+
+We expect multiple contributors to work in parallel where possible.
+
+## Describe your relationship to the maintainers of this technology. Are you yourself the maintainer? Do they know you plan to do this work and do they support it? Please tell us more about how you obtained their support and how you plan to work together to make sure your contributions are accepted.
+
+[1. Security tracker]
+
+I am not part of [security team](), but I am [contributor](https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+author%3ARaitoBezarius+label%3A%221.severity%3A+security%22+is%3Aclosed) and [reviewer of security fixes](https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+reviewed-by%3ARaitoBezarius+label%3A%221.severity%3A+security%22+is%3Aclosed+) in nixpkgs in critical software.
+and I am the [current release manager of NixOS](https://nixos.org/community/teams/nixos-release.html) and released NixOS 23.05, I had to deal with [Node.js]() and [OpenSSL]() [end of life status]()
```

Comment:
Add links or remove them, empty dead links are not a good option

Reply:
Yeah, I was doing that, it's still WIPy